src/utilities.ts

```diff
@@ -24,6 +24,15 @@ export function trimStart(target: string, trim: string): string {
   return target;
 }
 
+export function spacesValidation(text: string): boolean {
+  let trimmed = text.trim();
+  let noSpaces = true;
+
+  if (trimmed.indexOf(" ") > -1) noSpaces = false;
+
+  return noSpaces;
```

Comment:
Should we not trim the input text or does our service layer handle a space at the end/beginning of the branch name?

Reply:
The trailing space is not handled anywhere else and ADO complains that it is an invalid branch name if we do send it through. I had originally thought it would be nice to the space because I feel like its accidental most of the time when its at the end, but I can remove the trimming and just throw the error!